packages/liquidator/src/OneInchExchange.js

```diff
@@ -59,38 +84,55 @@ class OneInchExchange {
       });
     }
 
-    // 1 Split config
-    const flags = 0; // Enables all exchanges
-
-    // Number of pieces source volume could be splitted
-    // (Works like granularity, higly affects gas usage.
-    // Should be called offchain, but could be called onchain
-    // if user swaps not his own funds, but this is still considered
-    // as not safe)
-    // More info: https://github.com/CryptoManiacsZone/1inchProtocol#getexpectedreturn
-    const parts = 2;
-
     const expectedReturn = await this.oneSplitContract.methods
-      .getExpectedReturn(fromToken, toToken, amountWei, parts, flags)
+      .getExpectedReturn(fromToken, toToken, amountWei, this.oneInchParts, this.oneInchFlags)
       .call();
 
     const { returnAmount, distribution } = expectedReturn;
 
     this.logger.debug({
       at: "OneInchExchange",
       message: "GetExpectedReturn",
+      currentTime,
       returnAmount,
       distribution
     });
 
-    // TODO: Remove hardcoded gas
-    const tx = await this.oneSplitContract.methods
-      .swap(fromToken, toToken, amountWei, returnAmount, distribution, flags)
-      .send({ ...options, gasPrice, gas: 8000000 });
+    if (minReturnAmountWei && toBN(returnAmount.toString()).lt(toBN(minReturnAmountWei.toString()))) {
+      this.logger.warn({
+        at: "OneInchExchange",
+        message: "One Inch exchange return amount too low",
+        currentTime,
+        returnAmount,
+        minReturnAmountWei
+      });
+      return;
+    }
+
+    // Swap
+    const swapF = await this.oneSplitContract.methods.swap(
```

Comment:
Curious, why is this var named `swapF`? A bit confusing why you have the F

Reply:
The `f` notation to denote that its a partial function.

Reply:
I've never seen that before is that typical?

Reply:
`swap` is already a class member, which is why `swapF` was used. I can rename it to `swapPartialFunc` if its too confusing for you

Reply:
+1 `swapPartialFunc` is more readable